sites/frontend/components/Header/Nav/Pillars/index.js

```diff
@@ -18,17 +19,18 @@ const Pillars = styled('ul')({
 
 type Props = {
     showMainMenu: boolean,
+    pillars: Array<LinkType>,
 };
 
-export default ({ showMainMenu }: Props) => (
+export default ({ showMainMenu, pillars }: Props) => (
     <Pillars>
-        {pillarsConfig.map(pillar => (
+        {pillars.filter(pillar => pillar.title !== 'More').map(pillar => (
```

Comment:
Can we assume that 'more' will be the last thing in the pillar list rather than filtering the list every time we render the page? I guess there are only 5 things in the list so it's not a real problem...

Reply:
Yeah agree this feels pretty ugly to me.

Reply:
You could just map over `pillars` and in the callback function conditionally check whether `pillar.title === 'More'` If so you can `return '';` otherwise  ` return <Pillar />`.